packages/api-request-builder/test/default-params.spec.js

```diff
@@ -8,6 +8,7 @@ describe('defaultParams', () => {
     setDefaultParams.call({ features: serviceFeatures, params })
     expect(params).toEqual({
       id: null,
+      key: null,
```

Comment:
Functionality wise I think this works great. I'm just wondering if it makes sense to add it here since `key` is not included inside the below function? Should it be added to both, perhaps?
https://github.com/commercetools/nodejs/blob/a1560ec8819cbb84af4896929b35dddbf70a64c0/packages/api-request-builder/src/default-params.js#L13-L34

Reply:
@daern91 I thought about that too, but technically speaking the `key` param is not a query parameter: It's part of the uri and that's why I didn't want to add it to this function 😄

Reply:
Ahh, good point. Then we're ready to go :D